redash/query_runner/salesforce.py

```diff
@@ -76,6 +76,10 @@ def configuration_schema(cls):
                 },
                 "sandbox": {
                     "type": "boolean"
+                },
+                "api_version": {
+                    "type": "string",
+                    "title": "Salesforce API Version"
```

Comment:
Can you please add a "default" property with the value of `DEFAULT_API_VERSION`?

Thanks.